src/components/light.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Light = props => {
+    let onOrOff = props.isOn ? 'on' : 'off';
+    return (
+        <button
+            data-testid={props.testid}
+            className={'grid-item ' + onOrOff}
```

Comment:
The pattern we often use for conditional css classes leverages the [classnames](https://www.npmjs.com/package/classnames) package which is a tiny convenience helper for concatenating conditional css classes.

Usage looks like:

```jsx
<div
  className={cx('always-applied classes-go here', {
    on: props.isOn,
    off: !props.isOn
  })}
/>
```

The keys in the object you pass are added as classes if their corresponding value is truthy.

What you have is fine for this simple case though.